src/Blocks/AbstractBlocks.php

```diff
@@ -77,9 +64,10 @@ public function addThemeSupport(): void
 	 */
 	public function getBlocksDataFullRaw(): void
 	{
-		if (!$this->blocks) {
+		if (! $this->blocks) {
```

Comment:
```suggestion
		if (!$this->blocks) {
```

Not sure why linter isn't picking this up 🤷🏼‍♂️